src/lib.rs

```diff
@@ -67,7 +66,7 @@ macro_rules! report_error {
                     .build(),
             )
             .with_backtrace(&backtrace)
-            .send()
+            .send();
```

Comment:
somehow adding a semicolon changes the lifetime of some of the variables? the compiler told me to do it.

Reply:
I'm not very familiar with the macro side of Rust. Normally, adding a semicolon to the last expression of a function causes it to not be returned. If that behaves the same for macros, then this _would_ impact the lifetime of the value, since it would now live beyond the scope of the function.

As far as I can tell the macros defined in `./lib.rs` weren't very consistent about whether they returned the result of the `.send()` call or not. It seems fine to me to make them all consistent, one way or another.